tests/utils/basic.inc

```diff
@@ -1,8 +1,11 @@
 <?php
 require __DIR__ . "/" . "tools.php";
 
-
-$FILENAME = sys_get_temp_dir() . "/PHONGO-SERVERS.json";
+if (file_exists("{$_SERVER['HOME']}/PHONGO-SERVERS.json")) {
+	$FILENAME = "{$_SERVER['HOME']}/PHONGO-SERVERS.json";
```

Comment:
That feels a littlebit dodge. Like the idea though
How about environment variable? PHONGO_SERVERS=/path/to/PHONGO-SERVERS.json